front/components/assistant_builder/ActionsScreen.tsx

```diff
@@ -140,7 +137,7 @@ export function hasActionError(
     case "GITHUB_GET_PULL_REQUEST":
       return hasErrorActionGithub(action);
     case "REASONING":
-      return hasErrorActionReasoning(action);
+      return null;
```

Comment:
We kinda want to display an error if the config is not valid anymore, this is an edge case but if I tick the capability because my workspace had O1 and then my admin deactivates it, the action will be broken and so the agent? 

I'm guessing we archive assistant when they used a model that is deactivated, maybe we need something similar for the reasoning model tool of an agent?

Reply:
What i'll do: at same time, if there is no reasoning model or if reasoning is disabled, I'll strip the resoning config. This feels like a good tradeoff WDYT ?

Reply:
Yes this will just feel weird users might wonder where is that checkbox they can't see anymore (as opposed to having it greyed out understanding why you can't use it). But agreed it's an edge case and we can keep as is!